src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -137,43 +170,67 @@ function TransactionListItem<TItem extends ListItem>({
             text={transactionItem?.tag}
         />
     );
-
-    const taxCell = (
+}
+function TaxCell({showTooltip, amount, currency}: CurrencyCellProps): JSX.Element {
+    const styles = useThemeStyles();
+    return (
         <TextWithTooltip
             shouldShowTooltip={showTooltip}
-            text={CurrencyUtils.convertToDisplayString(taxAmount, currency)}
+            text={CurrencyUtils.convertToDisplayString(amount, currency)}
             style={[styles.optionDisplayName, styles.label, styles.pre, styles.justifyContentCenter, styles.textAlignRight]}
         />
     );
+}
+const CompareProps = (prevProps: CellProps, nextProps: CellProps) => prevProps.keyForList === nextProps.keyForList;
+const MemoDateCell = memo(DateCell, CompareProps);
```

Comment:
Can you combine `PureComponent` and `memo` like
```js
const Greeting = memo(function Greeting({ name }) {
  console.log("Greeting was rendered at", new Date().toLocaleTimeString());
  return <h3>Hello{name && ', '}{name}!</h3>;
}, arePropsEqual);
```

see https://react.dev/reference/react/PureComponent